crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -1860,7 +1873,7 @@ fn test_invalid_condition_args_terminator() {
     let spend = &conds.spends[0];
     assert_eq!(*spend.coin_id, test_coin_id(H1, H2, 123));
     assert_eq!(a.atom(spend.puzzle_hash).as_ref(), H2);
-    assert_eq!(spend.flags, ELIGIBLE_FOR_DEDUP | HAS_RELATIVE_CONDITION);
```

Comment:
I don't think your change is supposed to affect the DEDUP flag, right?

Reply:
Correct, I've reverted this